translate/hi_IN.po

```diff
@@ -0,0 +1,222 @@
+# Translation of menu11 in LANGUAGE
+# Copyright (C) 2021
+# This file is distributed under the same license as the menu11 package.
+# Telegram : @VCARP <viren.28opensky@gmail.com>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: menu11\n"
+"Report-Msgid-Bugs-To: https://github.com/prateekmedia/menu11\n"
+"POT-Creation-Date: 2021-04-09 22:46-0400\n"
+"PO-Revision-Date: 2021-10-04 16:30+0530\n"
+"Last-Translator: Virendra Carpenter <virendra carpenter>\n"
+"Language-Team: \n"
+"Language: hi_IN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../metadata.desktop
+msgid "Menu 11"
+msgstr "मेनू 11"
+
+#: ../metadata.desktop
+msgid "A highly customizable launcher inspired from windows 11 menu. Fork of MenuZ."
+msgstr "विंडोज़ 11 मेनू से प्रेरित एक उच्च अनुकूलन योग्य लांचर। मेनूजेड का कांटा।"
```

Comment:
MenuZ Ka kanta? Have you used direct google translate? 
Kanta doesnt resembles to fork in context to git.

Reply:
sorry for that

Reply:
check now it's okay